### ISSUE TITLE
The Documentation message cannot be dismissed via keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1843]
![Image](https://github.com/user-attachments/assets/84e81a1a-c43a-484a-bc28-c44f90aa989c)
This documentation message cannot be dismissed using the keyboard. Ideally, I should be able to focus on the close button and dismiss the notification.
